examples/hi_xiaowen/s0/README.md

```diff
@@ -8,3 +8,50 @@ FRRs with FAR fixed at once per hour:
 | DS_TCN(spec_aug)      | 287       | 80(avg30) | 0.008176   | 0.005075     |
 | MDTC                  | 156       | 80(avg10) | 0.007142   | 0.005920     |
 | MDTC_Small            | 31        | 80(avg10) | 0.005357   | 0.005920     |
+
+Next, we use CTC loss to train the model, with DS_TCN and FSMN.
+and we use CTC prefix beam search to decode and detect keywords,
+the detection is either in non-streaming or streaming fashion.
+
+Since the FAR is pretty low when using CTC loss,
+the follow result is FRRs with FAR fixed at once per 12 hours:
+
+Comparison between Max-pooling and CTC loss.
+The CTC model is fine-tuned with base model trained on WenetSpeech(23 epoch).
+FRRs with FAR fixed at once per 12 hours
+
+
+| model                 | loss        | hi_xiaowen | nihao_wenwen |
+|-----------------------|-------------|------------|--------------|
+| DS_TCN(spec_aug)      | Max-pooling | 0.051217   | 0.021896     |
+| DS_TCN(spec_aug)      | CTC         | 0.056574   | 0.056856     |
+
+
+Comparison between DS_TCN(Pretrained with Wenetspeech, 23 epoch)
```

Comment:
is it possible to release the pre-trained model, so people can reproduce the experiment?

Reply:
Pretrained models are released in the latest commit(README).